src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/AssociatedMetadataTypeTypeDescriptionProvider.cs

```diff
@@ -1,6 +1,9 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+// TODO: Enable after System.ComponentModel.TypeDescriptionProvider is annotated
+#nullable disable
+
```

Comment:
Can't put a comment there but in here:
`public override ICustomTypeDescriptor GetTypeDescriptor(Type objectType, object instance)`

should `object` be nullable (I'm not familiar with the API just shape and name of the API suggest like there should be no issue with providing a type but not providing instance and possibly also reverse)

Reply:
Am also not familiar with this API, so am waiting for it to be annotated by someone who hopefully does :)